var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -102,29 +103,56 @@ def install(self, spec, prefix):
             float_options.append('--enable-sse2')
```

Comment:
I can't comment on lines that you didn't change, but there's an `autoreconf` call up above. Might want to move that to the `autoreconf` stage.